FAIL
Kernel panic in run: PANIC at ../../tests/threads/tests.c:93 in fail(): test failed
Call stack: 0xc00285cc 0xc002a4d3 0xc002a430 0xc0020751
Translation of call stack:
0xc00285cc: debug_panic (.../../lib/kernel/debug.c:38)
0xc002a4d3: pass (.../tests/threads/tests.c:99)
0xc002a430: run_test (.../tests/threads/tests.c:57)
0xc0020751: run_actions (.../../../threads/init.c:341)
